src/components/simulation/tray.tsx

```diff
@@ -14,27 +16,26 @@ export const Tray: React.FC<IProps> = (props) => {
   const { trayAnimals, hidden, onHideTray } = props;
   return (
     <div className={`tray ${hidden ? "hidden" : ""}`}>
+      <SortingTray />
       <div className="header">
-        <div className="title">Leaf Pack Sorting Tray</div>
+        <div className="title">{t("SORTINGTRAY")}</div>
         <button className="close" onClick={onHideTray}>
           <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12" width="12" height="12">
```

Comment:
Why is the close button svg here instead of an image folder?

Reply:
It was easy to inline and I was following the DESE model that the orange team was using for these simple SVGs (they do the same thing with the thumbnail x close).  Probably worth having in an asset eventually - especially if we need to reuse it.

Reply:
I moved it into a file.  Now we can reuse it anywhere using the same pattern that we have for the other SVGs.